week4/teamBuilder.js

```diff
@@ -0,0 +1,52 @@
+//Below is just a reference file for my teamBuilder function
+//This is embedded inside my cohorts.js file
+function teamBuilder (strOfMembers, quantity, method){
```

Comment:
Should use this like a `utility` or `library` function.
Recommended to place this file within a `lib` or a `utils` folder`.
Then, at the bottom of the file, write:
```js
module.exports = teamBuilder;
```

And within another file, you can `require` this function from this file:

```js
const teamBuilder = require('../path/to/this/file/utils/teamBuilder');
```
And use the function there.